r/R/arrow-datum.R

```diff
@@ -139,17 +140,16 @@ Math.ArrowDatum <- function(x, ..., base = exp(1), digits = 0) {
     cumprod = eval_array_expression("cumulative_prod_checked", x),
     cummax = eval_array_expression("cumulative_max", x),
     cummin = eval_array_expression("cumulative_min", x),
+    cosh = eval_array_expression("cosh", x),
+    sinh = eval_array_expression("sinh", x),
+    tanh = eval_array_expression("tanh", x),
+    acosh = eval_array_expression("acosh_checked", x),
+    asinh = eval_array_expression("asinh", x),
+    atanh = eval_array_expression("atanh_checked", x),
```

Comment:
Mostly out of curiosity: did we miss these earlier? Or are they also new?

Reply:
They're new as of https://github.com/apache/arrow/pull/44630 which hasn't been released.